src/index.html

```diff
@@ -1,14 +1,42 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0,    minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <header class="header">
+      <img
+        class="icon"
+        src="images/logo.png"
+        alt="moyo"
+      >
+      <nav class="nav">
+        <div class="nav__list">
+          <a href="" class="nav__items is-active">apple</a>
+          <a href="" class="nav__items">samsung</a>
+          <a href="" class="nav__items">смартфоны и телефоны</a>
+          <a href=""
```

Comment:
```suggestion
          <a
            href="#"
```
make link destination "#" everywhere